metronet-profile-picture.php

```diff
@@ -109,7 +109,8 @@ public function ajax_get_thumbnail() {
 		$post_id = isset( $_POST[ 'post_id' ] ) ? absint( $_POST[ 'post_id' ] ) : 0;
 
 		if ( has_post_thumbnail( $post_id ) ) {
-			$post_thumbnail = get_the_post_thumbnail( $post_id, 'thumbnail' );
+			$thumb_src = wp_get_attachment_image_src( get_post_thumbnail_id($post_id), 'thumbnail' , false, '' );
```

Comment:
Arguments need to be separated with spaces. Ex: `get_post_thumbnail_id( $post_id )`